### PR TITLE
method name is "onReceive", not "receive"

### PR DESCRIPTION
**Pull request by cosmolev**

### CODE REVIEW

File: akka-samples/akka-sample-main-scala/tutorial/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 happens in the <code>preStart</code> method, where a <code>Greeter</code> actor is created
 and instructed to issue that greeting we crave for. When the greeter is done it
 will tell us so by sending back a message, and when that message has been
-received it will be passed into the behavior described by the <code>receive</code>
+received it will be passed into the behavior described by the <code>onReceive</code>
```
Comment (ktoso) [inline akka-samples/akka-sample-main-scala/tutorial/index.html]: In Scala it actually is `receive`, so this fix is not needed in this file IMO

**Comment by ktoso** [comment]
Thanks for the fix, in the java tutorials - it's a valid fix there.
Would you mind squashing these 3 commits into one (and exclude the change in `akka-sample-main-scala`), making the commit message align our [contributing guidelines](https://github.com/akka/akka/blob/master/CONTRIBUTING.md), so it would be like: `=sam method name is "onReceive", not "receive"`?

Thanks!

**Comment by ktoso** [comment]
Ping @cosmolev, can you please `git rebase --interactive` and squash these 3 commits into 1? 
We'll merge once that's done.

**Comment by cosmolev** [comment]
@ktoso done. Sorry for the delay.

**Comment by ktoso** [comment]
LGTM, thanks!

**Pull request merged**